plugin/src/main/kotlin/io/github/ttypic/swiftklib/gradle/task/CompileSwiftTask.kt

```diff
@@ -118,14 +118,154 @@ abstract class CompileSwiftTask @Inject constructor(
     }
 
     private fun createPackageSwift(dependencies: List<SwiftPackageDependency>) {
-        val packageSwiftContents = createPackageSwiftContents(cinteropName, dependencies)
+        execOperations.exec {
```

Comment:
I think the error ouput of a command should be visible by the user of the plugins in all case, it's really important for knowing what were wrong with swift spm.
But, it's currently not the case.